images/pull_kubernetes_bazel/Makefile

```diff
@@ -12,12 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = 0.12
+IMG = gcr.io/k8s-testimages/bazelbuild
+TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
 
-image:
-	docker build -t "gcr.io/k8s-testimages/bazelbuild:$(VERSION)" .
+all: build
 
-push:
-	gcloud docker -- push "gcr.io/k8s-testimages/bazelbuild:$(VERSION)"
+build:
```

Comment:
Manually today. We would like to automatically create these images at some sort of regular cadence, but we have yet to write that piece.